docs/guides/latest-updates.mdx

```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
+### Qiskit Runtime IBM Client 0.25.0
+
+*To see the release notes for all versions, visit the [Qiskit Runtime IBM Client release notes](/api/qiskit-ibm-runtime/release-notes).*
+
+**Highlights from the 0.25.0 (03 July 2024) version release**
+
+#### Deprecation notes
+
+- The `optimization_level` option in EstimatorV2 is deprecated. Instead, you can perform circuit optimization using the [Qiskit transpiler](transpile) or [Qiskit transpiler service](qiskit-transpiler-service).
+- Interim results have been deprecated, including the [`interim_results()`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.RuntimeJobV2#interim_results) and [`stream_results()`](api/qiskit-ibm-runtime/qiskit_ibm_runtime.RuntimeJobV2#stream_results) methods.
+
+#### New features
+
+- Jobs now have an [`instance`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.RuntimeJobV2#instance) property that returns the instance that was used for the job.
+- `default_shots` is now a supported option when using EstimatorV2 in [local testing mode](local-testing-mode).
+
+See the full [0.25.0 release notes](/api/qiskit-ibm-runtime/release-notes#0250-2024-07-02).
+{/* use url format /api/qiskit-ibm-runtime/release-notes#0XYZ to link to the specific 0.XY.Z version of the release notes */}
+
+
 ### Qiskit Runtime IBM Client 0.24.0
```

Comment:
Should this be removed?

Reply:
We're discussing that with @javabster

Reply:
Consensus: we'll keep the latest 3 months, to be parallel with the SDK releases